src/utils/sas9/extractUserNameSas9.ts

```diff
@@ -0,0 +1,14 @@
+export const extractUserNameSas9 = (response: string) => {
+  const regex = /"title":\s?"Log Off [0-1a-zA-Z ]*"/
+  const fallbackRegex = /"title":\s?"[0-1a-zA-Z ]*"/
+  const matched = response?.match(regex) || response?.match(fallbackRegex)
+  const username = matched?.[0].slice(17, -1)
```

Comment:
How about we do the following:
```js
let username = matched?.[0].replace(/"title":\s?/, '')

username = username.replace('Log Off ', 'logoff ')

return username.split(' ').slice(1).join(' ')
```
such approach returns valid result for all test cases

Reply:
I changed approach completely, check it out.